pkg/controller/jobs/job/job_controller.go

```diff
@@ -121,11 +121,11 @@ func (h *parentWorkloadHandler) queueReconcileForChildJob(object client.Object,
 }
 
 type Job struct {
-	batchv1.Job
+	*batchv1.Job
```

Comment:
Uhm... Instead of making this a pointer, adding a new function, `SetObject(obj client.Object)` might be better.
@alculquicondor @mimowo @kerthcet WDYT?

Reply:
My understanding of `func Object()` was incorrect. So, a pointer sounds better.